src/og/Globe.js

```diff
@@ -210,6 +211,7 @@ class Globe {
                         minSlope: options.minSlope
                     }),
                 new TouchNavigation(),
+                new KeyboardNavigation(),
```

Comment:
KeyboardNavigation should be optional. No need to include the control into defaults. Please remove it. If you want KeyboardNavigaion in your project you could add it manualy. If we add keyboard here it will broke a lot of deefault keyboards handlers in relative projects.

Reply:
Makes sense.  Done!

Reply:
Forget to remove from example in Globe, next time). Please, update often from master, because lint is fixed now, it argued with in KML. But KML is fixed now.